docs/rules/Moq1400.md

```
@@ -0,0 +1,47 @@
+| Item | Value |
+| --- | --- |
+| Enabled | True |
+| Severity | Warning |
+| CodeFix | False |
+---
+
+Mocks use the `MockBehavior.Loose` by default. Some people find this default behavior undesirable, as it can lead to
+unexpected behavior if the mock is improperly set up. To fix, specify either `MockBehavior.Loose` or
+`MockBehavior.Strict` to signify acknowledgement of the mock's behavior.
+
+
```

Markdownlint

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)